lazyscribe/repository.py

```diff
@@ -31,14 +31,12 @@ class Repository:
     fpath : str | Path, optional (default "repository.json")
         The location of the repository file. If no repository file exists, this will be the location
         of the output JSON file when ``save`` is called.
-    mode : {"r", "a", "w", "w+"}, optional (default "w")
+    mode : {"r", "w", "w+"}, optional (default "w")
         The mode for opening the repository.
 
-        * ``r``: Repository loaded as read-only: no new artifacts can be logged.
-        * ``a``: All existing artifacts will be loaded as
-          read-only and new artifacts can be added.
+        * ``r``: No new artifacts can be logged.
         * ``w``: No existing artifacts will be loaded.
-        * ``w+``: All artifacts will be loaded in editable mode.
+        * ``w+``: All artifacts will be loaded.
```

Comment:
Looking at this now, maybe the more precise verbiage is

```suggestion
        * ``w+``: All existing artifact metadata will be loaded.
```